app/src/main/java/com/pvp/app/ui/screen/calendar/TaskFormsEdit.kt

```diff
@@ -69,269 +70,240 @@ fun TaskEditDialog(
         else -> "Description"
     }
 
-    Dialog(onDismissRequest = onDialogClose) {
```

Comment:
Altered the same as with `TaskFormsCreate` file, except here are no task type selectors. So, only `when`, `if` changed to `if` chain and `Slider` swapped with `Picker` for `duration` and `reminderTime` properties.